### PR TITLE
simplify build process

### PR DESCRIPTION
**Pull request by davidchambers**
I've wanted to do this for some time. I can now run

```console
$ VERSION=0.17.1 make
```

to generate the various files for the website, which feels like the appropriate amount of effort. The [current process][1] involves several manual steps.

[1]: https://github.com/ramda/ramda/issues/894#issuecomment-97192690

### CODE REVIEW

File: jsdoc/templates/docs/index.html.handlebars [hunk 1]
```diff
@@ -122,9 +122,6 @@
         </section>
         {{/each}}
     </main>
-    <script>
-        var DOC_DATA = {{json data}};
-    </script>
```
Comment (davidchambers) [inline jsdoc/templates/docs/index.html.handlebars]: Why did we do this, does anyone know?

File: package.json [hunk 2]
```diff
@@ -2,53 +2,19 @@
...
-    "bootstrap": "~3.3.1",
-    "gitbook-cli": "~0.3.4",
-    "grunt": "~0.4.5",
-    "grunt-cli": "~0.1.13",
-    "grunt-contrib-copy": "^0.7.0",
-    "grunt-contrib-less": "^1.0.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-express": "^1.4.1",
-    "grunt-jsdoc": "^0.6.1",
-    "grunt-open": "^0.2.3",
```
Comment (davidchambers) [inline package.json]: :tada:

**Comment by CrossEye** [comment]
LGTM

I would like one day to replace the `make` implementation with Node code or NPM scripts, so that its OS-independent.  But I'm not doing so now, and this is clearly an improvement.

:herb:

File: package.json (hunk 2 above)
Comment (buzzdecafe) [inline package.json]: looks like a win to me

File: jsdoc/templates/docs/index.html.handlebars (hunk 1 above)
Comment (CrossEye) [inline jsdoc/templates/docs/index.html.handlebars]: I'm guessing it was a debugging tool accidentally left in, but I don't remember for sure.

**Comment by CrossEye** [comment]
I published the `0.18.0` docs with this, and they look great.

There are two caveats.  The first one is quite minor:  We had been publishing `0.15`, `0.16`, `0.17`, with `0.17.1`, etc. as needed.  This required me to publish `0.18.0`.  I don't think this is a big deal, but it's sort of too bad.

The second one is something that got lost that we really should get back.  I'm trying to get to bed earlier these days, so I'm not digging into it now, but if someone else wants to look into why, it would be nice to know:  We've been pointing http://ramdajs.com/docs/ at the current build's API docs.  I thought that had been automated by the build, but it didn't happen here.

**Pull request merged**